src/main/java/seedu/quotesify/store/Storage.java

```diff
@@ -85,10 +89,11 @@ public static void load() {
             fileReader.close();
         } catch (ParseException | IOException e) {
             // e.printStackTrace();
+            ListManager.initialiseAllLists();
```

Comment:
Isn't the list manager initialized before the load command is called in main

Reply:
Ah okay, initialize is called again when an error has occurred during updating in the even the data was written halfway

Reply:
Yup you're right this line can be removed 👍